perllib/FixMyStreet/DB/Result/Comment.pm

```diff
@@ -283,7 +283,7 @@ sub meta_line {
 
     my $contributed_as = $self->get_extra_metadata('contributed_as') || '';
     my $staff = $self->user->from_body || $self->get_extra_metadata('is_body_user') || $self->get_extra_metadata('is_superuser');
-    my $anon = $self->anonymous || !$self->name;
+    my $anon = $self->anonymous || !$self->name || $cobrand->call_hook('is_comment_anonymous', $self);
```

Comment:
I don't think you need to pass anything to the call_hook as it is unless this is something I don't understand - same in below call

Reply:
Yeah you're right. I was trying to predict the future, but if/when this is needed the problem/comment argument can be added then. Will remove.